server/NucleiClassification/NucleiClassification.py

```diff
@@ -0,0 +1,179 @@
+import os
+import sys
+import json
+import colorsys
+
+import numpy as np
+import pandas as pd
+from sklearn.externals import joblib
+import dask.dataframe as dd
+
+from ctk_cli import CLIArgumentParser
+
+import logging
+logging.basicConfig(level=logging.CRITICAL)
+
+sys.path.append(os.path.normpath(os.path.join(os.path.dirname(__file__), '..')))
+from cli_common import utils as cli_utils # noqa
+
+
+def gen_distinct_rgb_colors(n, seed=None):
+    """
+    Generates N visually distinct RGB colors
+
+    Parameters
+    ----------
+    n : int
+        Number of distinct RGB colors to output
+
+    seed: int or array_like, optional
+        Seed for the random number generator. See documentation of
+        `numpy.random.seed` for more details
+
+    Returns
+    -------
+    colors_list : list
+        A list of n RGB colors
+
+    References
+    ----------
+    .. [#] http://martin.ankerl.com/2009/12/09/how-to-create-random-colors-programmatically/  # noqa
```

Comment:
I don't think `# noqa` is needed here. `python_static_analysis_HistomicsTK_plugins` doesn't fail locally for me if it's removed.